VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:


BO_ 768 AMS_HEARTBEAT_ID: 7 Vector__XXX
 SG_ ams_heatbeat_state : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ ams_heartbeat_counter : 8|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 769 AMS_DATA_ID: 8 Vector__XXX
 SG_ ams_state : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ ams_error : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ imd_error : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ lowest_voltage : 24|8@1+ (1,0) [0|0] "" Vector__XXX




